frontend/components/App/StatusBar/Counter.tsx

```diff
@@ -0,0 +1,121 @@
+/*!
+ * This file is part of the Stooa codebase.
+ *
+ * (c) 2020 - present Runroom SL
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { useEffect, useState } from 'react';
+
+import { IConferenceStatus, ITimeStatus } from '@/jitsi/Status';
+import { Fishbowl } from '@/types/api-platform';
+import useTranslation from 'next-translate/useTranslation';
+
+interface Props {
+  fishbowlData: Fishbowl;
+  timeStatus: ITimeStatus;
+  conferenceStatus: IConferenceStatus;
+  isModerator: boolean;
+}
+
+export const Counter = ({ fishbowlData, timeStatus, conferenceStatus, isModerator }: Props) => {
+  const [completedTime, setCompletedTime] = useState<boolean>(false);
+  const [timeToDisplay, setTimeToDisplay] = useState<string>('Loading...');
+  const [intervalTimer, setIntervalTimer] = useState<NodeJS.Timeout>();
+  const [fishbowlDate, setfishbowlDate] = useState(() => {
+    return conferenceStatus === IConferenceStatus?.NOT_STARTED
+      ? Date.parse(fishbowlData.startDateTimeTz)
+      : Date.parse(fishbowlData.endDateTimeTz);
+  });
+
+  const { t } = useTranslation('fishbowl');
+
+  const checkSecondsToDate = (currentDate: number, fishbowlDate: number): number => {
```

Comment:
What do you think about using a more accurate name like `calculateDuration` ?  

Also the setCompletedTime(true) can be outside this function